Updated translations from Phrase

pix-service-prod opened the pull request:
Exported the most recent locale files from Phrase

pix-service-prod committed:
Updated translations from Phrase

Updated locale file en.json
Updated locale file fr.json
Created locale file nl.json

pix-bot-github commented:
Une fois les applications déployées, elles seront accessibles via les liens suivants :
  * [App (.fr)](https://app-pr7584.review.pix.fr)
  * [App (.org)](https://app-pr7584.review.pix.org)
  * [Orga (.fr)](https://orga-pr7584.review.pix.fr)
  * [Orga (.org)](https://orga-pr7584.review.pix.org)
  * [Certif (.fr)](https://certif-pr7584.review.pix.fr)
  * [Certif (.org)](https://certif-pr7584.review.pix.org)
  * [1D](https://1d-pr7584.review.pix.fr)
  * [Admin](https://admin-pr7584.review.pix.fr)
  * [API](https://api-pr7584.review.pix.fr/api/)
  * [Audit Logger](https://pix-audit-logger-review-pr7584.osc-fr1.scalingo.io/api/)

Les variables d'environnement seront accessibles via les liens suivants :
  * [scalingo front](https://dashboard.scalingo.com/apps/osc-fr1/pix-front-review-pr7584/environment)
  * [scalingo api](https://dashboard.scalingo.com/apps/osc-fr1/pix-api-review-pr7584/environment)
  * [scalingo audit-logger](https://dashboard.scalingo.com/apps/osc-fr1/pix-audit-logger-review-pr7584/environment)

pix-service-prod committed:
Updated translations from Phrase

Updated locale file en.json
Updated locale file fr.json
Updated locale file nl.json
